dvc/utils/serialize/_toml.py

```diff
@@ -1,9 +1,12 @@
+import logging
 from contextlib import contextmanager
 
 from funcy import reraise
 
 from ._common import ParseError, _dump_data, _load_data, _modify_data
 
+logger = logging.getLogger(__name__)
```

Comment:
This is unused, let's get rid of this? (I assume you were only using it for debugging).